packages/app/components/claim/claim-button.tsx

```diff
@@ -320,7 +319,7 @@ export const ClaimButton = ({
             tw="ml-1 text-sm font-semibold leading-5"
             style={{ color: isDark ? colors.black : colors.white }}
           >
-            {isAuthenticated ? "Pre-Save to Collect" : "Pre-Save on Spotify"}
+            Pre-Save on Spotify
```

Comment:
What about this line?

Reply:
yeah thought to make the copy a bit consistent. Pre-Save to collect actually pre-saves on Spotify so thought we should maybe change it. wdyt?